mne/source_estimate.py

```diff
@@ -3196,6 +3196,12 @@ def stc_near_sensors(evoked, trans, subject, distance=0.01, mode='sum',
         If True, project the electrodes to the nearest ``'pial`` surface
         vertex before computing distances.
     %(subjects_dir)s
+    src : instance of SourceSpaces | instance of SourceMorph
+        The source space. Can also be a SourceMorph to morph the STC to
+        a new subject (see Examples).
```

Comment:
I would not allow `SourceMorph` here, just `src`. People can morph ahead of time if they want.